proto/ibc/core/channel/v1/channel.proto

```diff
@@ -79,6 +81,18 @@ enum State {
   STATE_ACKUPGRADE = 7 [(gogoproto.enumvalue_customname) = "ACKUPGRADE"];
 }
 
+// FlushStatus defines the status of a channel end pertaining to in-flight packets.
+enum FlushStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // Default status, the channel is not in an upgrade handshake
+  FLUSH_STATUS_NOTINFLUSH_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "NOTINFLUSH"];
```

Comment:
should `UNSPECIFIED` / `UNINITIALIZED` be a separate field to `NOTINFLUSH`? I think its probably fine to treat them the same

Reply:
I think it is fine to treat them the same since the default (0 value) needs to be set after the opening handshake to ensure backwards compatibility (otherwise proofs on older versions would fail). It would be odd to me to place the channel in `UNINITIALIZED` until the first upgrade occurs and then in `NOTINFLUSH` after that. What do you think?

Reply:
Yep, I agree!